PWGHF/TableProducer/treeCreatorToXiPi.cxx

```diff
@@ -31,6 +31,7 @@ namespace o2::aod
 namespace full
 {
 // from creator
+DECLARE_SOA_COLUMN(CollisionCounter, collisionCounter, int);
```

Comment:
a warning from my side: in case you will build your small trees on a relatively large dataset, this counter might be not ideal id defined as `int`. Indeed, beware that the max. integer is 2147483647 (just a bit more than 2B)

Reply:
Thanks @mfaggin, I switched to long int, that should be enough, right?

Reply:
actually the limit I mention above is exactly for a `long int`, maybe it's better to go for `long long int`, if you really need it

Reply:
@vkucera  I did not think about it, but may it depend on the value specified for `Max DF size` in the derived data tab on the hyperloop wagon?

Reply:
guys you are right indeed, sorry, my fault